index.html

```diff
@@ -65,5 +62,218 @@ <h1 class = "top-heading"> Hi there! I'm Sameed</h1>
+
+        <article class="first-project">
+            <img src="./Images/2 multi-post stories.png" alt="" class="work-image">
+            <div class="details-container">
+                <div class="heading-and-highlights">
+                    <h1 class="project-heading">Multi-Post<br>Stories</h1>
+                    <ul class="highlights">
+                        <li class="project-type">CANOPY</li>
+                        <li><img src="./Images/Counter.png" alt="counter" class="counter1"></li>
+                        <li class="which-end">Back End Dev</li>
+                        <li><img src="./Images/Counter.png" alt="counter" class="counter2"></li>
+                        <li class="project-year">2015</li>
+                    </ul>
+                </div>
+                <div class="project-description">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </div>
+                <ul class="tags">
+                    <li class="html">html</li>
+                    <li class="css">css</li>
+                    <li class="javascript">javascript</li>
+                </ul>
+                <div class="project-button">
+                    <span class="project-text">
+                        See Project
+                    </span>
+                </div>
+            </div>
+        </article>
+
+        <article class="first-project">
+            <img src="./Images/3 tonic.png" alt="" class="work-image">
+            <div class="details-container">
+                <div class="heading-and-highlights">
+                    <h1 class="project-heading">Tonic</h1>
+                    <ul class="highlights">
+                        <li class="project-type">CANOPY</li>
+                        <li><img src="./Images/Counter.png" alt="counter" class="counter1"></li>
+                        <li class="which-end">Back End Dev</li>
+                        <li><img src="./Images/Counter.png" alt="counter" class="counter2"></li>
+                        <li class="project-year">2015</li>
+                    </ul>
+                </div>
+                <div class="project-description">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </div>
+                <ul class="tags">
+                    <li class="html">html</li>
+                    <li class="css">css</li>
+                    <li class="javascript">javascript</li>
+                </ul>
+                <div class="project-button">
+                    <span class="project-text">
+                        See Project
+                    </span>
+                </div>
+            </div>
+        </article>
+
+        <article class="first-project">
+            <img src="./Images/4 multi-post stories.png" alt="" class="work-image">
+            <div class="details-container">
+                <div class="heading-and-highlights">
```

Comment:
- [ ] Kindly note in the original design, the project cards have a background of `ffffff` while the whole section has a background color of `e5e5e5`. I would suggest that you bring out that contrast.

![image](https://user-images.githubusercontent.com/98453979/213707658-ac8bfd26-8332-4b81-a3aa-b9beebd3595f.png)

Reply:
when I set e5e5e5 as the background color, it changes the colour of the first page (intro page as well). When I set its background colour as white, the ellipses and other shape in the beckground get hidden behind the white color. How do I fix that?